packages/lib/models/settings/settingValidations.ts

```diff
@@ -19,7 +19,7 @@ const validations: Record<string, ValidationHandler> = {
 			const downloadErrorCount = await Resource.downloadStatusCounts(Resource.FETCH_STATUS_ERROR);
 			if (downloadErrorCount > 0) return _('Some attachments could not be downloaded. Please try to download them again.');
 
-			const disabledCount = await BaseItem.syncDisabledItemsCount(Setting.value('sync.target'));
+			const disabledCount = await BaseItem.syncDisabledItemsCount(Setting.value('sync.target'), true);
```

Comment:
```suggestion
			const disabledCount = await BaseItem.syncDisabledItemsCount(Setting.value('sync.target'), false);
```

Currently, it may not be clear that ignored items are preventing the user from switching sync targets.

Here are a few options:
- Don't make ignored items prevent switching sync targets. Note that this solution could be okay for upload issues (e.g. "item too large to upload"),  but could cause problems for if the item was disabled due to a download issue.
- Add a way to easily unignore all issues and attempt a resync.

Reply:
Resolved in https://github.com/laurent22/joplin/pull/10261/commits/7dfef999b099c3ab15c4c1d390e29087fcd65a74.